.storybook/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+const webpack = require("webpack");
+const {plugins, resolve} = require("../webpack.common");
+const path = require("path");
+
+module.exports = {
+  resolve,
+  module: {
+    loaders: [
+      {test: /\.json$/, loader: "json"},
+      {
+        test: /\.css?$/,
+        loaders: ['style', 'raw'],
+        include: path.resolve(__dirname, '../')
```

Comment:
The fact that ESLint isn't imploding on the unholy union of single and double quotes is proof that I have no idea what I'm doing. My guess is that ESLint is ignoring this folder because it starts with a ".". We may need to explicitly lint this folder as well in our lint task.